docs/readmes/howtos/inbound_roaming.md

```diff
@@ -50,22 +53,27 @@ you need:
 - Functioning [Or8cr](https://docs.magmacore.org/docs/orc8r/architecture_overview),
 - Install [Federatetion Gateway](https://docs.magmacore.org/docs/feg/deploy_intro) and,
 - Install [Access Gateway](https://docs.magmacoreorg/docs/lte/setup_deb).
-- Configure it as a [Federated Deployment](https://docs.magmacore.org/docs/feg/federated_FWA_setup_guide)
+- Create a Federate Deployment (see [below](#Create a Federated Deployment)).
 - Make sure your setup is able to serve calls with your local subscribers
-  (in case of using subscriber DB you will need to set `"hss_relay_enabled":
-  false` on Federated LTE Network temporally to test this. Set it back to
-  `true` once tested!!)
 
-At the end of this you will have:
-- A Federated LTE Network with an LTE Gateway (AGW)
-- A Federation Network with a Federation Gateway (FeG)
+Once you are done you should either:
+- Local Non Federated case:  `a Federated LTE Network with an LTE Gateway (AGW)` + `a Federation Network WITHOUT Federation Gateway (FeG)`
```

Comment:
I think we should specify that this means `feg_lte` network that manages the access gateway and a `feg` type network that manages the federation gateway.